tck/src/main/java/ee/jakarta/tck/data/framework/junit/anno/Transaction.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+package ee.jakarta.tck.data.framework.junit.anno;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import ee.jakarta.tck.data.framework.junit.extensions.TransactionConditionExtension;
+
+/**
+ * These are tests that require Transactions for testing. 
+ * This annotation will verify that Transaction is available.
+ * If Transaction is not available, tests are skipped, otherwise, tests are run.
+ */
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+@ExtendWith(TransactionConditionExtension.class)
+public @interface Transaction {
```

Comment:
I don't think we need this.  The Jakarta EE Transactions spec is part of web profile, and that is the only place we need to be concerned about testing interoperability with it.  Other usage would be outside of the spec.